CMakeLists.txt

```diff
@@ -53,7 +57,7 @@ if(EVMC_TESTING)
     enable_testing()
 endif()
 
-if(EVMC_TESTING OR EVMC_TEST_TOOLS)
+if(EVMC_TESTING OR EVMC_TOOLS)
```

Comment:
Here `EVMC_TOOLS` check would be enough as `EVMC_TESTING` implies `EVMC_TOOLS`. I just left it for clarity. The end goal is:

```
if(EVMC_TOOLS)
    add_subdirectory(tools)
endif()

if(EVMC_TESTING)
    add_subdirectory(test)
endif()
```